app/views/reports/standard_form_report/_form_summary_subset.html.erb

```diff
@@ -0,0 +1,151 @@
+                  </td>
+                </tr>
+
+                <%# if no items, show the 'no data' message %>
+                <% if summary.items.length == 0 %>
+                  <tr class="summary">
+                    <td colspan="<%= result_cols + 2 %>">
+                      <em class="no_match"><%= I18n.t('report/report.no_match') %></em>
+                    </td>
+                  </tr>
+
+                <%# otherwise, loop over each item (answer) and make a row for it %>
+                <% else %>
+                  <% summary.items.each do |item| %>
+                    <tr class="summary">
+                      <td colspan="<%= result_cols + 2 %>">
+                        <%= render("reports/standard_form_report/form_summary_item", item: item) %>
+                      </td>
+                    </tr>
+                  <% end %>
+
+                <% end %>
+
+              <% else %>
+                <tr class="summary">
+                  <td class="num">#<%= summary.questioning.full_dotted_rank %></td>
+                  <td class="qname">
+                    <%= render("reports/standard_form_report/form_summary_question_label", summary: summary) %>
+                  </td>
+
+                  <% # if flow type, or if any other type and items are empty, just print the responses sep'd by commas %>
+                  <% if cluster.display_type == :flow || summary.items.length == 0 %>
+                    <td colspan="<%= result_cols %>">
+                      <% if summary.items.length == 0 %>
+                        <em class="no_match"><%= t("report/report.no_match") %></em>
+                      <% end %>
+
+                      <%= summary.items.map{|item| "#{item.text}"}.join(", ") %>
+                    </td>
+
+                  <% # if structured type with less than result_cols, do separate cells %>
+                  <% elsif cluster.display_type == :structured && cluster.headers.length <= result_cols %>
+                    <% summary.items.each do |item| %>
+                      <td class="structured"><%= render("reports/standard_form_report/form_summary_item", item: item) %></td>
+                    <% end %>
+
+                    <% # print out blank cell to fill up to max %>
+                    <% if result_cols > cluster.headers.length %>
+                      <td colspan="<%= result_cols - cluster.headers.length %>"></td>
+                    <% end %>
+
+                  <% # if structured type with more than result_cols, do one big cell %>
+                  <% elsif cluster.display_type == :structured && cluster.headers.length > result_cols %>
+
+                    <td colspan="<%= result_cols %>">
+                      <%= summary.items.each_with_index.map{|item, idx| render("reports/standard_form_report/form_summary_item_with_header", item: item, header: cluster.headers[idx]) }.join(", ") %>
```

Comment:
I am pretty sure you're going to have problems here with html_safe stuff because you're calling `join` on strings that could have html in them. See `safe_join`.